rules/builds.smk

```diff
@@ -311,18 +311,21 @@ rule tree:
         "logs/tree_{region}.txt"
     benchmark:
         "benchmarks/tree_{region}.txt"
-    threads: 4
+    threads: 8
     resources:
         # Multiple sequence alignments can use up to 40 times their disk size in
         # memory, especially for larger alignments.
         # Note that Snakemake >5.10.0 supports input.size_mb to avoid converting from bytes to MB.
-        mem_mb=lambda wildcards, input: 40 * int(input.size / 1024 / 1024)
+        mem_mb=lambda wildcards, input: 40 * int(input.size / 1024 / 1024),
+    params:
+        mem_gb=lambda wildcards, resources: int(resources.mem_mb / 1024)-1,
     conda: config["conda_environment"]
     shell:
         """
         augur tree \
             --alignment {input.alignment} \
             --output {output.tree} \
+            --tree-builder-args="-mem {params.mem_gb}G" \
```

Comment:
This doesn't work as expected (at least, not on AWS) where we get 
```
Job FAILED after 290.4 minutes (OutOfMemoryError: Container killed due to memory usage, exited 1)
```
Despite with iq-tree being run with `-mem 30G` and snakemake & AWS given 40Gb.

Reply:
Seems worth trying a node size with a much larger memory allocation, making sure the rule is benchmarked, and then seeing what is actually used to guide node size choice.

Reply:
Will do -- running now on a node with 64Gb (confusingly, `params.mem_gb` is 32, not sure why).